index.html

```diff
@@ -19,7 +20,7 @@
           </header>
       <main class="section2">
```

Comment:
- [ ] kindly consider checking on this [requested change](https://github.com/gisachris/week1_portfolio/pull/2/files#r1148415692) from the previous reviewer and make sure you fixed it to make your work more professional